js/ui/widget/ui.errors.js

```diff
@@ -218,6 +218,46 @@ module.exports = errorUtils(errors.ERROR_MESSAGES, {
     */
     E1052: '{0} should have the "dataSource" option specified',
 
+    /**
+    * @name ErrorsUIWidgets.E1053
+    */
+    E1053: "The 'buttons' option must include an object or a string items only",
+
+    /**
+    * @name ErrorsUIWidgets.E1054
+    */
+    E1054: "Text Editor button's 'location' property can be 'after' or 'before' only",
+
+    /**
+    * @name ErrorsUIWidgets.E1055
+    */
+    E1055: "Text Editor button must have a name",
+
+    /**
+    * @name ErrorsUIWidgets.E1056
+    */
+    E1056: "Text Editor button's 'name' field must be a string",
+
+    /**
+    * @name ErrorsUIWidgets.E1057
+    */
+    E1057: "The 'buttons' item must have unique name, but the \"{0}\" value has duplicates",
+
+    /**
+    * @name ErrorsUIWidgets.E1058
+    */
+    E1058: "The predefined '{0}' button must have no 'location' or 'options' fields in the configuration",
+
+    /**
+    * @name ErrorsUIWidgets.E1059
+    */
+    E1059: "Editor does not support '{0}' action button",
```

Comment:
What editor?

Reply:
This error should be generated if user want to use "spins" button for SelectBox for example.
Maybe "The "{0}" button type is not supported for this type of editor."